pkg/proof/proof_test.go

```diff
@@ -221,3 +221,35 @@ func TestNewShareInclusionProof(t *testing.T) {
 		})
 	}
 }
+
+// TestAllSharesInclusionProof creates proof for all the shares in a block.
+// Since we can't prove multiple namespaces at the moment, the block will contain all the shares with the same namespace.
```

Comment:
[nit] rephrased + wrapped 

```suggestion
// TestAllSharesInclusionProof creates a proof for all shares in the data
// square. Since we can't prove multiple namespaces at the moment, all the
// shares use the same namespace.
```